docs/self-managed/operational-guides/update-guide/820-to-830.md

```diff
@@ -105,3 +105,22 @@ function returned `false` if the variable didn't exist.
 
 With the new version, the behavior changed if the variable value is `null`. The function returns `true` if the variable
 exists and its value is not `null`. It returns `false` if the variable doesn't exist or its value is `null`.
+
+## Multi-tenancy
+
+### Zeebe migration
+
+Upgrading may take longer if the experimental consistency checks enabled. If the duration of the downtime needs to
```

Comment:
I believe we should use "update" instead of upgrade

Reply:
Nice catch!